pyro/contrib/bnn/utils.py

```diff
@@ -0,0 +1,16 @@
+import torch
```

Comment:
nit: from future import ...

By ensuring all files use the same programming language, we make it safer to debug and refactor by moving code among files.

Reply:
done